chapters/synchronous.tex

```diff
@@ -1319,10 +1333,10 @@ \subsection{Solver Methods}\label{solver-methods}
 This data is tool specific and is typically either defined with a vendor annotation or is given in the simulation environment.
 \end{nonnormative}
 
-\subsection{Associating a Solver to a Partition}\label{associating-a-solver-to-a-partition}
+\subsection{Associating a Solver to a Sub-Partition}\label{associating-a-solver-to-a-partition}
```

Comment:
I suppose it is fine to associate solvers also to discrete-time sub-partitions, even though they only apply to discretized ones?